### DISCUSSION TITLE
Need help with IOM Creation

### DISCUSSION THREAD

**Discussion by Idonotuse** [idea-statica/ideastatica-public#757] [Rhino/Grasshopper] [answered]
Hi Nathan, 

I couldn't build an IOM from the imported idea statica file. 

![image](https://github.com/user-attachments/assets/1ebf2a6e-ee4a-45aa-8153-5bbbf5fff231)

The same issue persists when I try to build IOM by importing xml file.

![image](https://github.com/user-attachments/assets/100acf89-4019-45f7-bb08-4b40fd35673d)

Can you let me know what the issue could be? Thanks in Advance.

Additional Info:
[V24_1.zip](https://github.com/user-attachments/files/18228567/V24_1.zip) : sample xml file used to import.
Idea statica versions: V23.1, V24.0 and V24.1 (24.1.0.3002)
grasshopper plugin (V0.1.13)
Rhino 8

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi @Idonotuse, we are currently in the process of updating Rhino to the new version of IOM. If you created the .ideacon in version 23.1 you should ensure the API is opened from version 23.1. By default it will always load the latest version of API which is why you will always get error.

To do it you need to go into the rhino package folder for the plugin. 

%appdata%\McNeel\Rhinoceros\packages\8.0\IDEAStatiCa\0.1.13\

Open IdeaConnection.dll.config file. Change the directory to the version.

![image](https://github.com/user-attachments/assets/36ffdc92-7f2f-4639-8641-c576b9cd46e3)

We will let you know when the error in the latest version is fixed.
